src/plugins/discover/public/application/components/default_discover_table/helper.tsx

```diff
@@ -83,12 +90,14 @@ export function getLegacyDisplayedColumns(
   if (!Array.isArray(columns) || typeof indexPattern !== 'object' || !indexPattern.getFieldByName) {
     return [];
   }
+  // TODO: Remove overrides once we support PPL/SQL sorting
```

Comment:
did we want to create an issue and track it on github and link a reference in here?